crates/aggchain-proof-builder/src/config.rs

```diff
@@ -41,3 +47,7 @@ impl Default for AggchainProofBuilderConfig {
 fn default_aggchain_prover_timeout() -> Duration {
     Duration::from_secs(3600)
 }
+
+fn default_aggregated_proof_vkey_hash() -> B256 {
+    hex!("00441b614a713401ad1090cd7e59813e07352cb247172934f5a05dedb9e671bf").into()
```

Comment:
Is it the real default value?

Reply:
The vkey is now a constant and it just refers to the value from `aggchain_proof_core`.